src/_pytest/_code/code.py

```diff
@@ -489,8 +489,8 @@ def fill_unfilled(self, exc_info: Tuple["Type[_E]", _E, TracebackType]) -> None:
         assert self._excinfo is None, "ExceptionInfo was already filled"
         self._excinfo = exc_info
 
-    @property
-    def type(self) -> "Type[_E]":
+    @property  # noqa: A003
```

Comment:
Ditto here for `property`

Reply:
resolving as per my comment below your review. Please reopen if you disagree